### ISSUE TITLE
Archive this project and point at https://github.com/dequelabs/axe-core-npm ?

### ISSUE THREAD

**Issue by karlhorky** [dylanb/react-axe#187]
First of all, thanks so much for your tireless work in the accessibility community - really amazing!

Since this is now deprecated, would you consider archiving this GitHub repo and then pointing at the new one at https://github.com/dequelabs/axe-core-npm ?

**Comment by stephenmathieson** [comment]
We had this https://github.com/dequelabs/axe-core-npm/issues/4. Looks like we didn't finish it?

**Comment by stephenmathieson** [comment]
@straker @WilcoFiers bump

**Comment by dylanb** [comment]
@straker @WilcoFiers bump

**Comment by straker** [comment]
I agree that we should archive, but I don't have access rights to do it.
